templates/python/setup.py.twig

```diff
@@ -6,6 +6,7 @@ setuptools.setup(
   version = '{{sdk.version}}',
   license='{{spec.licenseName}}',
   description = '{{sdk.shortDescription}}',
+  long_description = '{{sdk.readme}}',
```

Comment:
The `readme` property is not populated from the Appwrite side, let's use the `gettingStarted` content instead

```suggestion
  long_description = '{{sdk.gettingStarted}}',
```